### PR TITLE
Pcc52

### PR DESCRIPTION
**Pull request by abkosar**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [5]
Estimated time spent (hours): [2]
Completed (yes/no): [yes]
I stretched my coding skills (if yes what did you learn?): [Of course! This was Day 3 of 100 Days of Code in Python. I learned to use timedeltas.]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 52/abkosar/timer.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    i = 0
+    print(timer.seconds)
+    while i < timer.seconds:
+        os.system("clear")
+        print(timer)
+        timer = timer - one_second
+        i += 1
+        time.sleep(1)
+
+    print("Your timer is completed!")
```
Comment (feoh) [inline 52/abkosar/timer.py]: Super teeny nit - according to flake8/pep8 there should be a newline at end of file.

File: 52/abkosar/timer.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import argparse
+import time
+import os
+from datetime import datetime, timedelta
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Pomodoro Timer')
```
Comment (feoh) [inline 52/abkosar/timer.py]: You might consider refactoring your argument parsing into its own function.

File: 52/abkosar/timer.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                        required=True)
+
+    args = parser.parse_args()
+    timer_duration = args.duration
+    timer = timedelta(minutes=timer_duration)
+    one_second = timedelta(seconds=1)
+
+    i = 0
+    print(timer.seconds)
+    while i < timer.seconds:
```
Comment (feoh) [inline 52/abkosar/timer.py]: You might consider using something like a for loop and the range() built-in function here in order for your code to be more pythonic,

File: 52/abkosar/timer.py (hunk 2 above)
Comment (abkosar) [reply 52/abkosar/timer.py]: Great advice, definitely will do that!

File: 52/abkosar/timer.py (hunk 1 above)
Comment (abkosar) [reply 52/abkosar/timer.py]: True, will fix it.

File: 52/abkosar/timer.py (hunk 3 above)
Comment (abkosar) [reply 52/abkosar/timer.py]: Should I fix these and do another PR?

**Pull request merged**